eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -558,6 +564,10 @@ public MutableShortBag collectShort(ShortFunction<? super V> shortFunction)
         return this.delegate.groupByEach(function, new UnifiedSetMultimap<VV, V>());
     }
 
+    /**
+     * @deprecated in 7.1. Use {@link OrderedIterable#zip(Iterable)} instead.
+     */
+    @Deprecated
```

Comment:
Same as above, let's separate this to another commit. Update version# to 8.0, too.